### ISSUE TITLE
Repro recipe: pin transformers<5 to load released checkpoints (fixes #68) — PushT 90% on MPS

### ISSUE THREAD

**Issue by rbaileyquintar** [lucas-maes/le-wm#80]
Posting a working reproduction recipe — on a fresh install the released checkpoints fail to load purely because `transformers` is unpinned.

**Root cause.** `stable_pretraining.vit_hf` builds a stock `transformers.ViTModel`. transformers 5.x renamed the ViT attention params:
- 4.x (what `weights.pt` was saved with): `encoder.encoder.layer.N.attention.attention.query…`
- 5.x: `encoder.layers.N.attention.q_proj…`

Neither `stable-worldmodel` nor `stable-pretraining` upper-bounds transformers (`>=4.50.0`), so a fresh install pulls 5.x and `load_pretrained` rebuilds an encoder whose keys no longer match the checkpoint → the `_target_`/state_dict mismatch in #68.

**Fix — pin transformers <5:**
```
pip install 'transformers==4.57.6'
```
Stock `load_pretrained('pusht/lewm')` then loads cleanly — no `_target_` stripping, no manual key mapping. Also needed `pip install hdf5plugin` (cf. #76) for the `hdf5` format to register.

**Result.** PushT, eval `seed=42`, Apple Silicon / MPS: **45/50 = 90%**, in line with the paper (96 ± 2.83) and #62 (98). The few-point gap is plausibly MPS-vs-CUDA numerics.

**Suggestion.** Add `transformers<5` to the `[train]`/`[env]` extras until the checkpoints are re-exported for 5.x.
